rotkehlchen/api/v1/schemas.py

```diff
@@ -2059,10 +2075,17 @@ class BlockchainAccountsPutSchema(BlockchainAccountsPatchSchema):
     ...
 
 
-class BlockchainAccountsDeleteSchema(AsyncQueryArgumentSchema):
-    blockchain = BlockchainField(required=True, exclude_types=(SupportedBlockchain.ETHEREUM_BEACONCHAIN,))  # noqa: E501
+class StringAccountSchema(Schema):
     accounts = fields.List(fields.String(), required=True)
 
+
+class BlockchainAgnosticAccountsDeleteSchema(ChainTypeSchema, StringAccountSchema):
+    """Used to manage accounts in different chains filtering by chain type"""
+
+
+class BlockchainAccountsDeleteSchema(StringAccountSchema, AsyncQueryArgumentSchema):
```

Comment:
consistency

```suggestion
class BlockchainTypeAccountsDeleteSchema(StringAccountSchema, AsyncQueryArgumentSchema):
```

Reply:
This is not for the blockchainType. This is the schema for deleting an account for a single blockchain. In the diff it looks strange because I extended it for other schemas but is not new